editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5705,7 +5717,9 @@ void CanvasItemEditorViewport::_perform_drop_data() {
 		if (res.is_null()) {
 			continue;
 		}
-		Ref<PackedScene> scene = Ref<PackedScene>(Object::cast_to<PackedScene>(*res));
+		Ref<PackedScene> scene = Object::cast_to<PackedScene>(*res);
+		Ref<Texture2D> texture = Object::cast_to<Texture2D>(*res);
+		Ref<AudioStream> audio_stream = Object::cast_to<AudioStream>(*res);
 		if (scene != nullptr && scene.is_valid()) {
```

Comment:
```suggestion
		if (scene.is_valid()) {
```
It should be like this.